docs/Config_Reference.md

```diff
@@ -4896,6 +4896,38 @@ data_ready_pin:
 #vref:
 #   The selected voltage reference. Valid values are: 'internal', 'REF0', 'REF1'
 #   and 'analog_supply'. Default is 'internal'.
+#fir_filter:
```

Comment:
The manual says: _"Only use these bits together with the 20-SPS setting in normal mode and the 5-
SPS setting in duty-cycle mode. Set to 00 for all other data rates."_

So the only valid setting to use this with is 20 SPS (this code doesn't allow duty cycle mode). If that's not the case this should raise an error.

This might still be useful to someone that wants 20SPS mode for filament scales.